src/citrine/_rest/resource.py

```diff
@@ -46,6 +48,49 @@ def access_control_dict(self) -> dict:
         }
 
 
+GEMDSelf = TypeVar('GEMDSelf', bound='GEMDResource')
+
+
+class GEMDResource(Resource[GEMDSelf]):
+    """A reference to a resource by UID."""
+
+    @classmethod
+    def build(cls, data: dict) -> GEMDSelf:
+        """Convert a raw, nested dictionary into Objects."""
+        if "context" in data and len(data) == 2:
```

Comment:
Which situations does this apply to?

Reply:
https://github.com/CitrineInformatics/gemd-python/blob/b625aee87d02768d918ccb4088a97b43fa15c894/gemd/json/gemd_json.py#L32
gemd-python makes a material history with a `object` key for the root object of the serialized graph
https://github.com/CitrineInformatics/citrine-python/blob/9be3cded48ea64f754d1897513100fd2230d0c52/src/citrine/resources/material_run.py#L141
platform returns a material history with a `root` key for a LinkByUID to the root object.